spec/services/reviewers_service_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ReviewersService, type: :service do
+  describe '.save_review' do
+    let!(:contribution) { create(:contribution) }
+
+    context 'when the attributes are valid' do
+      it 'updates reviewed_by attributes' do
+        ReviewersService.save_review(contribution.id, contribution.user)
+        expect { contribution.reload.reviewed_by }.to change {
+                                                        contribution.reviewed_by
+                                                      }.from(nil).to(contribution.user.name)
+      end
+
+      it 'updates reviewed_at attributes' do
+        mock_time = Time.current.round(6)
+        allow(Time).to receive(:current).and_return(mock_time)
+
+        ReviewersService.save_review(contribution.id, contribution.user)
+        contribution.reload
+        expect(contribution.reviewed_at).to eq(mock_time)
```

Comment:
essa forma não está errada, mas uma outra forma de fazer também é usando o travel_to

```rb
mock_time = Time.current
travel_to(mock_time) do
  ReviewersService.save_review(contribution.id, contribution.user)
  contribution.reload
  expect(contribution.reviewed_at).to eq(mock_time)
end
```

Reply:
Nice!  I'll use this solution. Thank u :fist_oncoming: